README.md

```diff
@@ -57,27 +60,24 @@ $ python ticker.py
 
 ### Configuration
 ```bash
-$ python ticker.py --help
-usage: ticker.py [-h] [--pairs_toml PAIRS_TOML] [--horizon_host HORIZON_HOST]
-                 [--time_duration TIME_DURATION]
-                 [--bucket_resolution BUCKET_RESOLUTION]
-                 [--output_file OUTPUT_FILE]
+$ python ticker.py -h
+usage: ticker.py [-h] [-c PAIRS_TOML] [-u HORIZON_HOST] [-t TIME_DURATION]
+                 [-bt BUCKET_RESOLUTION] [-o OUTPUT_FILE]
```

Comment:
it's usually good to keep just 1 char for single-dash args unless it's not possible. that's the expectation so it's confusing if we deviate from that.